Reset window insets for some apps and hide status bar

ceiba1985 opened the pull request:
1. Some app's UI display abnormally in the secondary display windows, seems 
    some activity layouts of the apps have dependency on the status bar. Reset
    the windowinsets of the secondary displays to incorporate the height of status bar.
2. Set the status bar's height to 0.

Tracked-On: OAM-100827
Tracked-On: OAM-100860
Tracked-On: OAM-101404
Signed-off-by: ji, zhenlong z <zhenlong.z.ji@intel.com>

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:Trigger Builds cancelled

More details, logs, and binaries: /absp/builders/celadon_rmr0_stable-preintegration/builds/312

ceiba1985 committed:
Reset window insets for some apps and hide status bar

1. Some app's UI display abnormally in the secondary display windows, seems
some activity layouts of the apps have dependency on the status bar. Reset
the windowinsets of the secondary displays to incorporate the height of status bar.
2. Set the status bar's height to 0.

Signed-off-by: jizhenlo <zhenlong.z.ji@intel.com>
